### ISSUE TITLE
"Already in community" warning is displayed on restart if user closes the app while joining to community

### ISSUE THREAD

**Issue by EmiM** [TryQuiet/quiet#1433]

**Comment by holmesworcester** [comment]
Moving to next sprint because we have bigger issues to fix in this sprint.

**Comment by kingalg** [comment]
I don't think that we have this problem anymore. We fixed a lot of issues with "already in community" modal in recent sprint. If @EmiM and @holmesworcester are not against it I would closes this and move to done.
